.gitlab-ci.yml

```diff
@@ -26,8 +25,14 @@ build-repos:
       - $CI_PROJECT_DIR/bsg_cadenv
       - $CI_PROJECT_DIR/basejump_stl
       - $CI_PROJECT_DIR/bsg_manycore
-      - $CI_PROJECT_DIR/bsg_f1
+      - $CI_PROJECT_DIR/bsg_bladerunner
     policy: push
+  only:
+    refs:
+      - merge_requests
```

Comment:
I don't think this will work.  Merge requests means gitlab merge requests, not github pull requests

Reply:
Oh! Worth a try :) I'm trying out few things...

Reply:
Yeah, it's a pain.  I think the way you have to do it is setting up a webhook on the GitHub where it pings the GitLab when a PR is created